posts/2019-12-03-survey-launch.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "Launching the 2019 State of Rust Survey"
+author: The Rust Community Team
+description: "Hearing from you about the fourth year of Rust"
+---
+
+It's that time again! Time for us to take a look at how the Rust project is doing, and what we should plan for the future. The Rust Community Team is pleased to announce our [2019 State of Rust Survey][survey]! Whether or not you use Rust today, we want to know your opinions. Your responses will help the project understand its strengths and weaknesses and establish development priorities for the future.
+
+Completing this survey should take about 10–15 minutes and is anonymous unless you choose to give us your contact information. We will be accepting submissions until December 16th, and we will write up our findings a month or so afterwards to [blog.rust-lang.org]. You can also check out see [last year’s results][2018 survey].
+
+- [English][survey]
+- [Simplified Chinese]
+- [Traditional Chinese]
+- [French]
+- [German]
+- [Italian]
+- [Japanese]
+- [Korean]
+- [Portuguese]
+- [Russian]
+- [Spanish]
+- [Swedish]
+- [Vietnamese]
+
+(If you speak multiple languages, please pick one)
+
+Please help us spread the word by sharing the survey link on your social network feeds, at meetups, around your office, and in other communities.
+
+If you have any questions, please see our [frequently asked questions] or email the Rust Community team at [community-team@rust-lang.org].
+
+Finally, we wanted to thank everyone who helped develop, polish, and test the survey. In particular we'd like to thank [Dr. Jasun Carr], for providing their expertise in communication processes with helping to create this year's survey, and all of the volunteers who worked to provide all of the translations available this year.
+
+[dr. jasun carr]: https://isu.edu/cmp/faculty-and-staff/facultyandstaff/d-jasun-carr.html
+[survey]: https://docs.google.com/forms/d/1iGnf8Mmf4JRggOJ3E7iZlBLsgeLxIYzaI1caiFHQ6OQ/
```

Comment:
This points to the master copy, it probably should not?

Reply:
Either that, or the forms document needs to be renamed.

Reply:
Updated the title, and removed the fixme.